ui-components/src/global/headers.scss

```diff
@@ -19,29 +19,21 @@
 }
 
 .detail-header__image {
-  @apply absolute;
+  @apply me-2;
```

Comment:
```suggestion
  @apply me-2;
  @apply ms-2;
```

Reply:
We lost some of the padding between the browser and the image, and this should bring it back